src/lib/Contracts.ts

```diff
@@ -138,7 +137,7 @@ export class Contract {
     path: string[] = [],
     casperClient?: CasperClient,
     stateRootHash?: string
-  ): Promise<StoredValue> {
+  ): Promise<any> {
```

Comment:
this is pain, but for now we won't do anything better, right?

Reply:
yeah, you are right